mobile-widgets/qml/TemplateTitle.qml

```diff
@@ -0,0 +1,26 @@
+// SPDX-License-Identifier: GPL-2.0
+import QtQuick 2.11
+import QtQuick.Controls 2.4
+import QtQuick.Layouts 1.11
+
+
+Item {
+	property alias title: myLabel.text
+	width: parent.width -20
+	height: myLabel.height + 1
+
+	Label {
+		id: myLabel
+		color: subsurfaceTheme.textColor
+		font.pointSize: subsurfaceTheme.titlePointSize
+		font.bold: true
+		anchors.horizontalCenter: parent.horizontalCenter
+		text: "ups"
```

Comment:
I realize that a user of the template should overwrite the text... so if you want to use the text set here to remind the developer of that, maybe have a slightly more instructive text `"using the TemplateTilte you need to set the title text"`

Reply:
changed.